client/signup/jetpack-connect/plans.jsx

```diff
@@ -12,7 +12,7 @@ import { bindActionCreators } from 'redux';
 import { getPlansBySite } from 'state/sites/plans/selectors';
 import { getFlowType } from 'state/jetpack-connect/selectors';
 import Main from 'components/main';
-import ConnectHeader from './connect-header';
+import StepHeader from '../step-header';
```

Comment:
Are imports ordered alphabetically? Searching devdocs for `import` is not very fruitful.